spec/std/isa/isa/globals.isa

```diff
@@ -2147,6 +2153,13 @@ function stage1_page_walk {
         }
       }
 
+      if (!implemented?(ExtensionName::Svrsw60t59b)) {
+        if ((PTESIZE >= 64) && pte[60:59] != 0) {
+          # 60:59 are reserved if Svrsw60t59b is not supported
+          raise (page_fault_code, effective_mode, vaddr);
+        }
+      }
```

Comment:
Similarly, shall we combine this with the earlier reserved bit test, or do the intervening checks need to be in a particular order?